docs/components/organization/organization-list.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: <OrganizationList />
+description: The `<OrganizationList />` component is used to display organization related memberships, invitations, and suggestions for the user.
+---
+
+# `<OrganizationList />`
+
+<Images
+  alt="Organization List Example"
+  width={992}
+  height={568}
+  src="/docs/images/ui-components/component-org_list.svg"
+/>
```

Comment:
@mzhong9723  Does this image exist ? I couldn't get it to work locally

Reply:
Added image as a PNG, not sure if we should be using a SVG to be consistent - if so, do we usually export those from Figma?

Reply:
Left a review comment, but these should be optimized, preferably as WebP.
I use Squoosh, and the default WebP settings.